src/rawdog/utils.py

```diff
@@ -88,3 +88,7 @@ def render_prompt(self):
             last_commit=self.last_commit,
             retries=self.retries,
         )
+
+
+def is_finetuned_model(model: str):
+    return "ft:" in model or "rawdog" in model or "abante" in model
```

Comment:
If the model has ft: in it we probably don't want to change the base url.

Reply:
Good point